share/errmsg-utf8.txt

```diff
@@ -18527,8 +18527,8 @@ ER_PLACEHOLDER_50021
 ER_PLACEHOLDER_50022
   eng "Placeholder"
 
-ER_PLACEHOLDER_50023
-  eng "Placeholder"
+ER_GIT_HASH
```

Comment:
Why this particular one? 50023?

Reply:
The same id (1523) was used in the 5.6 commit.